pylace/lace/engine.py

```diff
@@ -992,7 +997,19 @@ def update(
         ...         StateTransition.view_alphas(),
         ...     ],
         ... )
+
+        Use a common set of transitions by name, specifically "sams":
+
+        >>> engine.update(
+        ...     100,
```

Comment:
is this very fast in the doc test? we can knock this down to 5 or whatever if needed.

Reply:
Reduced. There's no harm if it's even faster now.